pages/account/cards.tsx

```diff
@@ -157,8 +167,84 @@ function CardsDisplay() {
             const theData = appContext.data.getData();
             setCardsData(theData.get("cards"));
             cardSets.forEach(cardSet => { cardSet.cards = (cards) ? cards.filter(card => card.data.category == cardSet.cardSetName) : [] });
+
+            const filterOptions = cards.filter(card => card.displayName != "New Monster?").map(card => {
```

Comment:
Removing the `cardSets` from the `useEffect` dependancy array solves the constant re-render problem, and we don't need to depend on it so that's fine.

However, I am finally noticing what you guys mean by the page being "laggy" or "slow" and I still haven't found a way to improve it .. maybe there's little we can do with so much information being rendered .. not sure.

I tried moving a lot of the calculations to the "pre-calculation" stage but that made little difference.